extensions/wikia/Search/WikiaSearchController.class.php

```diff
@@ -533,17 +559,18 @@ protected function getSearchConfigFromRequest() {
 		$searchConfig = new Wikia\Search\Config();
 		$resultsPerPage = $this->isCorporateWiki() ? self::INTERWIKI_RESULTS_PER_PAGE : self::RESULTS_PER_PAGE;
 		$resultsPerPage = empty( $this->wg->SearchResultsPerPage ) ? $resultsPerPage : $this->wg->SearchResultsPerPage;
-		$searchConfig
-			->setQuery                   ( $request->getVal( 'query', $request->getVal( 'search' ) ) )
-			->setCityId                  ( $this->wg->CityId )
-			->setLimit                   ( $request->getInt( 'limit', $resultsPerPage ) )
-			->setPage                    ( $request->getInt( 'page', 1 ) )
-			->setRank                    ( $request->getVal( 'rank', 'default' ) )
-			->setHub                     ( $request->getBool( 'hub', false ) )
-			->setInterWiki               ( $this->isCorporateWiki() )
-			->setVideoSearch             ( $request->getBool( 'videoSearch', false ) )
-			->setFilterQueriesFromCodes  ( $request->getArray( 'filters', [] ) )
-			->setBoostGroup              ( $request->getVal( 'ab' ) );
+		$searchConfig->setQuery( $request->getVal( 'query', $request->getVal( 'search' ) ) )
+			->setCityId(
+				$this->wg->CityId
+			)
+			->setLimit( $request->getInt( 'limit', $resultsPerPage ) )
+			->setPage( $request->getInt( 'page', 1 ) )
+			->setRank( $request->getVal( 'rank', 'default' ) )
+			->setHub( $request->getBool( 'hub', false ) )
+			->setInterWiki( $this->isCorporateWiki() )
+			->setVideoSearch( $request->getBool( 'videoSearch', false ) )
+			->setFilterQueriesFromCodes( $request->getArray( 'filters', [] ) )
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "filters" 5 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)